internal/batches/executor/executor.go

```diff
@@ -168,15 +169,15 @@ func (x *executor) do(ctx context.Context, task *Task, ui TaskExecutionUI) (err
 
 	// Actually execute the steps.
 	opts := &RunStepsOpts{
-		Task:        task,
-		Logger:      l,
-		WC:          x.opts.Creator,
-		EnsureImage: x.opts.EnsureImage,
-		TempDir:     x.opts.TempDir,
-		IsRemote:    x.opts.IsRemote,
-		GlobalEnv:   x.opts.GlobalEnv,
-		Timeout:     x.opts.Timeout,
-		RepoArchive: repoArchive,
+		Task:              task,
+		Logger:            l,
+		WC:                x.opts.Creator,
+		EnsureImage:       x.opts.EnsureImage,
+		TempDir:           x.opts.TempDir,
+		GlobalEnv:         x.opts.GlobalEnv,
+		Timeout:           x.opts.Timeout,
+		RepoArchive:       repoArchive,
+		WorkspaceFilesDir: x.opts.WorkingDirectory,
```

Comment:
In this file, there are two new fields on two structs, but for the same thing. Should they be named the same then, to make it easier to trace what is coming from where?